engine/flink/schemed-kafka-components-utils/src/main/java/pl/touk/nussknacker/engine/schemdkafka/flink/typeinfo/ConsumerRecordTypeSerializerSnapshot.java

```diff
@@ -0,0 +1,46 @@
+package pl.touk.nussknacker.engine.schemdkafka.flink.typeinfo;
```

Comment:
@coderabbitai found a typo bug here. BTW why it is in java?

Reply:
It should be `schemedkafka` instead of `schemdkafka`

Reply:
Oups, I'll change it :) And like @coderabbitai wrote: java because of compatibility and I was suggesting by ScalaOptionSerializerSnapshot, which is also written in Java.

Reply:
Package is still wrong...